### PR TITLE
Add Schedule a Workflow in TypeScript and Python

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?
Adds samples for Python and TypeScript. 

TW-283
TW-356

## Notes to reviewers

https://github.com/temporalio/features/issues/153
PHP and Java aren't currently supported.

### CODE REVIEW

File: docs-src/python/how-to-set-a-schedule-in-python.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: How to set a Schedule in Python
+sidebar_label: Schedule
+description: Set a Schedule in Python for scheduling Workflows to be run at specific times in the future.
+tags:
+  - python
+  - developer-guide
+  - how-to
+---
+
+Use the [create_schedule()](https://python.temporal.io/temporalio.client.Client.html#create_schedule) method available in the [Client](https://python.temporal.io/temporalio.client.Client.html) class to create a new Schedule for a Workflow Execution
```
Comment (bergundy) [inline docs-src/python/how-to-set-a-schedule-in-python.md:12]:
```suggestion
Use the [create_schedule()](https://python.temporal.io/temporalio.client.Client.html#create_schedule) method available in the [Client](https://python.temporal.io/temporalio.client.Client.html) class to create a new Schedule for a Workflow Execution.
```

File: docs-src/python/how-to-set-a-schedule-in-python.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Define the parameters of the Schedule, then call the `create_schedule()` method on the Client object and pass in the Workflow Id for the Workflow you want to schedule, as well as the Schedule object you created.
```
Comment (bergundy) [inline docs-src/python/how-to-set-a-schedule-in-python.md:14]: Where's the action in this sample?
And note that both the workflow and schedule have IDs.

File: docs-src/python/how-to-set-a-schedule-in-python.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+spec=ScheduleSpec(
+            calendars=[
+                ScheduleCalendarSpec(
+                    second=(ScheduleRange(1, step=1),),
+                    minute=(ScheduleRange(2, 3),),
+                    hour=(ScheduleRange(4, 5, 6),),
+                    day_of_month=(ScheduleRange(7),),
+                    month=(ScheduleRange(9),),
+                    year=(ScheduleRange(2080),),
+                )
```
Comment (bergundy) [inline docs-src/python/how-to-set-a-schedule-in-python.md:27]: Please split this into multiple samples, one for cron, one for running on a specific interval and one using the calendar.
I'd also add comments about these fields and mention what defaults they have.

File: docs-src/python/how-to-set-a-schedule-in-python.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            ],
+            intervals=[
+                ScheduleIntervalSpec(
+                    every=timedelta(days=10),
+                    offset=timedelta(days=2),
+                )
```
Comment (bergundy) [inline docs-src/python/how-to-set-a-schedule-in-python.md:33]: This part specifically requires an explanation.

File: docs-src/typescript/how-to-set-a-schedule-in-typescript.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      // or a single datetime:
+      // calendars: [
+      //   {
+      //     comment: '1/1/23 at 9am',
+      //     year: 2023,
+      //     month: 1,
+      //     dayOfMonth: 1,
+      //     hour: 9,
+      //   },
+      // ],
```
Comment (bergundy) [inline docs-src/typescript/how-to-set-a-schedule-in-typescript.md:53]: This is clearer than the Python sample IMHO.

File: docs/application-development/features.md [hunk 6]
```diff
@@ -3434,6 +3434,118 @@ export async function sleepWorkflow(): Promise<void> {
...
+<Tabs
+defaultValue="go"
+queryString="lang"
+values={[{label: 'Go', value: 'go'},{label: 'Java', value: 'java'},{label: 'PHP', value: 'php'},{label: 'Python', value: 'python'},{label: 'TypeScript', value: 'typescript'},]}>
+
+<TabItem value="go">
+
+Content is planned but not yet available.
+
+The information you are looking for may be found in the [legacy docs](https://legacy-documentation-sdks.temporal.io/).
```
Comment (bergundy) [inline docs/application-development/features.md:3450]: Can we not direct users to the legacy docs?

File: docs-src/typescript/how-to-set-a-schedule-in-typescript.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+      type: 'startWorkflow',
+      workflowType: reminder,
+      args: ['♻️ Dear future self, please take out the recycling tonight. Sincerely, past you ❤️'],
+      taskQueue: 'schedules',
+    },
+    scheduleId: 'sample-schedule',
+    policies: {
+      catchupWindow: '1 day',
+      overlap: ScheduleOverlapPolicy.ALLOW_ALL,
+    },
```
Comment (bergundy) [inline docs-src/typescript/how-to-set-a-schedule-in-typescript.md:32]: Requires some explanation.

**Review by bergundy — COMMENTED** [review]
Is there going to be a follow up PR(s) showing how to update, list, describe, and delete schedules?

**Review by flossypurse — CHANGES_REQUESTED** [review]
I would like us to focus on using snipped examples from the samples repo.

**Pull request closed without merging**